apps/utils/cipher/cipher.main.kts

```diff
@@ -0,0 +1,462 @@
+#!/usr/bin/env kotlin
+
+@file:DependsOn("org.jetbrains.kotlinx:kotlinx-collections-immutable-jvm:0.3.5")
+@file:DependsOn("com.amazonaws:aws-encryption-sdk-java:2.4.0")
+@file:DependsOn("software.amazon.awssdk:kms:2.20.74")
+@file:DependsOn("com.squareup.okio:okio:3.3.0")
+
+import com.amazonaws.encryptionsdk.AwsCrypto
+import com.amazonaws.encryptionsdk.CryptoResult
+import com.amazonaws.encryptionsdk.kmssdkv2.KmsMasterKey
+import com.amazonaws.encryptionsdk.kmssdkv2.KmsMasterKeyProvider
+import kotlinx.collections.immutable.PersistentMap
+import kotlinx.collections.immutable.persistentMapOf
+import okio.*
```

Comment:
If we're sticking with our java standards in Kotlin, we should avoid star imports. (Not sure if Kotlin handles imports more intelligently where this isn't needed)